### PR TITLE
Change assessments to code reviews

### PR DESCRIPTION
**Pull request by dustinbrownman**
This is a big one. It changes a lot of files. If it goes through, you should make sure to merge these changes as soon as possible so any new code doesn't conflict with what's here. That said, all the changes center around the "code review" system (`code_reviews`, `submissions`, `objectives`, `reviews`, `grades`, and `scores`) and shouldn't affect anything else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is app handles a few different things for Epicodus students and staff, incl
 
 * Tuition
 * Attendance
-* Assessments
+* CodeReviews
```
Comment (michaelrkn) [inline README.md]: "Code reviews"

**Comment by michaelrkn** [comment]
Fix the readme and merge!

**Comment by jenmccarthy** [comment]
Some of the students in the PHP class are not able to access the epicenter site, perhaps because of this merge?

**Comment by dustinbrownman** [comment]
Shoot, I think I tracked this down what the problem is. Working on it now!

**Comment by dustinbrownman** [comment]
Well that was a fun bug to fix. I did a find and replace to change all instances of "Assessments" to "CodeReviews". I quickly realized that wasn't a good idea, and thought I had caught all the places that had been affected. But alas, there was still the migration file. I fixed that in 4cf52a0ab1ac18148542160afb92570cb109c39a and remigrated, but there was still a problem.

It refused to rename the `requirements` table to `objectives`. Turns out, the index that the requirements primary key was never renamed. Originally, requirements were called "assessment_requirements". That was changed not long after through a migration, but for some reason, the primary key index never changed. So when this migration happened, and it tried to `ALTER INDEX "requirements_pkey" RENAME TO "objectives_pkey"`, it broke, because the `requirements_pkey` was really `assessment_requirements_pkey`. Had to dig in PSQL to finally find that this was the case.

So note if you end up having migration errors when merging this code. You may have to run `ALTER INDEX "assessment_requirements_pkey" RENAME TO "requirements_pkey"` is PSQL to get it to work.

If anyone has a real explanation for WHY this happened, I'm all ears!

**Pull request merged**